prototypes/index.js

```diff
@@ -478,6 +496,18 @@ const breweryPrompts = {
     // Write your annotation here as a comment
   },
 
+  getSingleBreweryBeerCount(breweryName) {
```

Comment:
Also super similar to the above function...Again, might be useful for students to see.